update pullsecret for robot account

varodrig opened the pull request:
- Updated Devspaces to be installed using a catalog
- Update variable name for robot account when installing rhdh
- argocd roles added PipelineRun
- added a json file for the rhdh home page.

varodrig committed:
update pullsecret for robot account

varodrig committed:
update pullsecret name

varodrig committed:
update secret

varodrig committed:
update config argo

varodrig committed:
update var

varodrig committed:
remove ""

varodrig committed:
new devspaces implementation

varodrig committed:
update home page rhdh

varodrig committed:
update url

varodrig committed:
move file

varodrig committed:
update url

varodrig committed:
new folder for json

varodrig committed:
update json sintax based on rhdh version

varodrig committed:
deleting operator group

varodrig commented:
Needs a merge. @treddy08

varodrig commented:
Or anyone else if it can review/approve and merge please. @newgoliath  thanks

varodrig commented:
@juliaaano

juliaaano commented on rhdh/deploy/templates/app-config-rhdh.yaml:
```diff
@@ -105,6 +105,15 @@ data:
+      '/developer-hub':
+        target: https://raw.githubusercontent.com/
+        pathRewrite:
+          '^/api/proxy/developer-hub': /varodrig/agnosticd_workload_helm_charts/main/rhdh/json/rhdh_home.json
```
The path /varodrig is concerning, shouldn't it point to the upstream?

juliaaano commented on devspaces/values.yaml:
```diff
@@ -1,2 +1,26 @@
+  # Which role to grant the user on the namespace
+  role: admin
+
+# Deploy MTA Operator
```
It is not  really MTA here, is it?

juliaaano reviewed (commented):
I am not a user or developer of this helm chart, therefore I will abstain to approve, reject or merge it.
I am not sure if merging it now can cause an impact on RH1.
Furthermore, there are too many distinct changes which makes it more difficult for me to confidently assess the impact.

varodrig committed:
updates

varodrig commented:
- Updated Devspaces to be installed using a catalog
- Update variable name for robot account when installing rhdh (approved by @treddy08 )
- argocd roles added PipelineRun
- added a json file for the rhdh home page.

varodrig committed:
update url/name

varodrig committed:
update name
